Automattic/wp-calypso
# HelpCenter: center icon

## Proposed Changes

Adjust viewport of icon to center with/without notification bubble

| Notification | Normal |
| - | - |
| <img width="52" alt="Markup 2022-06-14 at 20 32 53" src="https://user-images.githubusercontent.com/33258733/173664239-8ff7bb48-d804-4006-a781-de231242c8a8.png"> | <img width="105" alt="Markup 2022-06-14 at 20 34 08" src="https://user-images.githubusercontent.com/33258733/173664297-4870d82a-0969-4a21-b547-9d4536f96e01.png"> |


## Testing Instructions

Pull branch and sync from editing-toolkit. You can toggle the notification by making the check negative in `packages/help-center/src/components/help-icon.tsx` on line 7.

Closes #64603
